src/client/pages/Payment/Payment.tsx

```diff
@@ -23,7 +22,13 @@ export const PaymentPage = () => {
   const textKeys = useTextKeys()
   const { path: pathLocale } = useCurrentLocale()
   const redirectURL = useStoredRedirectURL()
-  const [, trustlyURL] = useCreateTrustlyURL()
+
+  const [, trustlyURL] = useCreateTrustlyURL({
+    onError: () => {
```

Comment:
Should we add some logs for errors as well?

Reply:
taken care of inside the hook